src/main/java/com/example/TournamentProject/TournamentManagement.java

```diff
@@ -0,0 +1,77 @@
+package com.example.TournamentProject;
+
+import org.springframework.stereotype.Service;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.ArrayList;
+import java.util.List;
+
+@Service
+public class TournamentManagement {
+
+    public void getTournamentName() {
+        System.out.println("Enter tournament name:");
+    }
+
+    public String createTournamentName() {
+        String tournamentName = null;
```

Comment:
Why you wrote `= null` ???
What is default value for string?

**Java compiler assigns null as default value to String variable.**